src/webgl/3d_primitives.js

```diff
@@ -256,6 +253,14 @@ p5.prototype.box = function(width, height, depth, detailX, detailY) {
  *   background(205, 105, 94);
  *   rotateY(millis() / 1000);
  *   sphere(40, detailX.value(), 16);
+ *   describe(`A white rotating sphere-like shape with a diameter of 40.
+ *     Its actual shape is similar to a clam shell.
+ *     Black lines on its surface show how the sphere is built out of
+ *     many small triangles.
+ *     When adjusting the slider below the drawing, the shape increases
+ *     the number of small triangles making up the sphere, making the
+ *     final shape increasingly round as the slider approaches its
+ *     maximum setting.`);
```

Comment:
This is a very long description. Can we try to make it more succinct?

Reply:
As I tried to fulfill this request, I was able to rewrite this into something that I think is an improvement in _quality_, but isn’t much shorter.  

The challenges for a short description in this demo (and other similar demos) are:

1. The description of the main shape isn’t the whole story.  The entire point of this (and similar) demo(s) is to show how a big shape is built out of smaller shapes. Thus, for the description to include all the relevant info, it needs to include **both** the overall shape as well as the smaller triangles that comprise it.  
2. The description also needs to describe the slider (outside of the drawing), and how that slider’s value affects the drawing itself.  

These aren’t conditions that most of the other demos have. But since this demo is all about how the level of detail impacts both the visual and the performance, it seems to me that leaving that information out is omitting crucial information.

Do you think this is sufficient to justify a slightly longer description?  

(To be clear: I still think I can come up with a rewrite that is an improvement—just not substantially shorter.)